hack/v1migration/phase2.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -e
+
+if [ $# != 1 ]; then
+	echo "USAGE: $0 [WORKDIR]"
+	exit 1
+fi
+
+WORKDIR=$1
+echo "Using workdir: $WORKDIR"
+
+oc project hive
+
+oc scale -n hive deployment.v1.apps/hive-operator --replicas=0
+oc scale -n hive deployment.v1.apps/hive-controllers --replicas=0
+oc scale -n hive deployment.v1.apps/hiveadmission --replicas=0
+
+if [[ ! -d "$WORKDIR" ]]
+then
+    echo "Directory $WORKDIR does not exist."
+    exit 1
+fi
+
+
+HIVE_TYPES=( checkpoints clusterdeployments clusterdeprovisionrequests clusterimagesets clusterprovisions clusterstates dnsendpoints dnszones hiveconfig selectorsyncidentityprovider selectorsyncset syncidentityprovider syncsetinstance syncset )
+for i in "${HIVE_TYPES[@]}"
+do
+	:
+	echo "Deleting all ${i}"
+	oc delete ${i}.hive.openshift.io -A --all
+done
+
+oc delete customresourcedefinition.apiextensions.k8s.io/checkpoints.hive.openshift.io
```

Comment:
Why is this not being done in an array loop over HIVE_TYPES?

Reply:
That did not occur to me, will update it.